### PR TITLE
fixing broken links in Kommander

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-75402 

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials/index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ account and attach credentials to it.
     <p class="message--note"><strong>NOTE: </strong>Bitbucket Server does not support skipping TLS
     certificate verification for webhooks. If you use a self-signed certificate in your cluster, you
     must [add the certificate to Bitbucket Server](https://confluence.atlassian.com/bitbucketserver/if-you-use-self-signed-certificates-938028692.html).
-    Or if you are using Konvoy, you could [set up a Let's Encrypt certificate](https://docs.d2iq.com/dkp/konvoy/latest/security/letsencrypt/).</p>
+    Or if you are using Konvoy, you could [set up a Let's Encrypt certificate](https://docs.d2iq.com/dkp/kommander/1.4/security/).</p>
```
Comment (cneth) [inline pages/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials/index.md:103]: It doesn't seem to me like this new link really helps solve the problem here. That Kommander page doesn't have any information on how to set up a Let's Encrypt certificate.   

Also, this text explicitly mentions 'Konvoy' so it seems like this should still link back to the Konvoy docs.  Also, there *is* still documentation in the Konvoy docs about how to use Let's encrypt to create a certificate, it's here:  https://docs.d2iq.com/dkp/konvoy/1.7/access-authentication/letsencrypt/#create-a-lets-encrypt-certificate

To me, that's a better link, even though that subsection is just a piece of a bigger procedure in the Konvoy docs.

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials/index.md:103]: Ok, I'm at a loss as to why this link will not work then. Hmmm. Gonna have to dig in further.

Comment (msdolbey) [inline pages/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials/index.md:103]:
```suggestion
    Or if you are using Konvoy, you could [set up a Let's Encrypt certificate](/dkp/konvoy/1.8/access-authentication/letsencrypt/).</p>
```

**Comment by cmastr** [comment]
starting over

**Pull request closed without merging**